api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/instance/InstanceRefreshService.java

```diff
@@ -42,6 +50,18 @@
     private final CachedProductFamilyService cachedProductFamilyService;
     private final CachedServicesService cachedServicesService;
     private final InstanceRetrievalService instanceRetrievalService;
+    private boolean isStarted = false;
+
+    @InjectApimlLogger
+    private final ApimlLogger apimlLog = ApimlLogger.empty();
+
+    /**
+     * Starts refreshing the API Catalog cache
+     */
+    public void start() {
+        this.isStarted = true;
+        log.info("InstanceRefreshService started");
+    }
```

Comment:
Does it make sense to define also `INFO` messages and use the new apimlLogger instead of `Slf4j`?

Reply:
In the new logger, messages have to be coded and documented. The only useful message is service started. Other messages stay but are filtered. From the code perspective we decided to not change info into debug. Instead, original logback INFO messages are filtered and in debug mode, they are still info. You have the option to configure the logs back to as they were and keep the info-debug distinction.